app/models/Window/mainWindowRenderer.js

```diff
@@ -101,6 +101,45 @@ function enableWindowButtonListeners() {
     });
 }
 
+function scrollIntoView(elem, relativeTo) {
+    let pos = elem.offsetTop;
+    let top = typeof relativeTo !== "undefined" ? relativeTo.scrollTop : elem.parentNode.scrollTop;
+    let bottom = top + relativeTo.offsetHeight;
+    if (pos > bottom) {
+        elem.scrollIntoView(false);
+    } else if (pos < top) {
+        elem.scrollIntoView(true);
+    }
+}
+
+function enableKeyListeners() {
+    document.addEventListener("keydown", (e) => {
+        if (
+            e.keyCode === 38 || // up
+            e.keyCode === 40 // down
+        ) {
+            // prevent regular scrolling behaviour
+            e.preventDefault();
+
+            let eventId = document.querySelector("#events-container li.selected").id;
+
+            // Get all events in view.
+            // Not the same as all stored events, because we might be filtering.
+            let events = Array.from(document.querySelectorAll("#events-container li")).filter(
+                (elem) => elem.style.display !== "none"
+            );
+            let index = events.findIndex((elem) => elem.id === eventId);
+            let targetIndex = e.keyCode === 38 ? index - 1 : index + 1;
```

Comment:
If there are no predefined constants for these key codes, consider defining them yourself.

Reply:
Unfortunately, those constants do not exist. They can be mapped to their string counterparts, which at least makes it more readable.